Library/Homebrew/linuxbrew-core-migration.rb

```diff
@@ -0,0 +1,129 @@
+# typed: false
+# frozen_string_literal: true
+
+# List of formulae that had a revision in linuxbrew-core
+# that was different than in homebrew-core
+# We will use this list to modify the version_sheme
+# during the migration from linuxbrew-core to homebrew-core.
+
+# TODO: finish to fill the list
+LINUXBREW_CORE_MIGRATION_LIST = %w[
+  afuse
+  apng2gif
+  apr-util
+  archivemount
+  argon2
+  arpack
+  asciinema
+  atdtool
+  augustus
+  avfs
+  bandcamp-dl
+  cairo
+  ccls
+  codemod
+  cquery
+  credstash
+  csvtomd
+  cvs
+  cxxtest
+  datetime-fortran
+  ddd
+  diceware
+  docbook2x
+  docker-squash
+  doitlive
+  double-conversion
+  exif
+  expect
+  fades
+  fastme
+  fbi-servefiles
+  fftw
```

Comment:
This should go away on your next core merge.

Reply:
I will align the final list with the reality just before we merge this PR and a new release is cut.